### DISCUSSION TITLE
show hidden folders in file dialogs?

### DISCUSSION THREAD

**Discussion by 1e1001** [ppy/osu#15759] [Q&A] [answered]
my osu is installed in `~/.local/share/osu-stable/` and i can't import from there since osu doesn't show any hidden folders in the file prompt

i guess you could also fix this by adding a system dialog option or allowing users to type in file paths, but i think just showing hidden files or have a toggle somewhere would be the best solution

**Comment by bdach** [comment]
i could have sworn hidden files are hidden on explicit request, but i cannot find the appropriate pull request that would cover this.

so if this is to be supported then it should probably be a toggle.

**Reply by peppy** [reply]
Either a toggle, or shown with a lower opacity would work for me. I think in most of the osu! side usages, seeing hidden files would be more useful than not.

**Comment by Galm007** [comment]
I also have this problem. I use Artix linux and I installed the `osu` and `osu-lazer` packages from the Arch User Repository. My osu-stable folder is in `~/.local/share/osu-stable/`.

**Comment by Galm007** [comment]
nvm I got a temporary solution. What I did was I created a symlink for `~/.local/share/osu-stable` (the path to your osu! folder) and then put the link in a location where it doesn't require opening hidden folders to get into.

For example:
`ln -s ~/.local/share/osu-stable ~/osu-stable`

Then when you import data from osu-stable to osu-lazer, just select the symlink `~/osu-stable`.

**Comment by frenzibyte** [comment, marked as answer]
Tracking in https://github.com/ppy/osu/issues/18573
